### PR TITLE
Add Pauli expansion to GateOperation

### PR DESCRIPTION
**Pull request by viathor**

### CODE REVIEW

File: cirq/ops/gate_operation.py [hunk 1]
```diff
@@ -101,6 +101,9 @@ def _decompose_(self) -> op_tree.OP_TREE:
                                                     self.qubits,
                                                     NotImplemented)
 
+    def _pauli_expansion_(self) -> value.LinearDict[str]:
+        return protocols.pauli_expansion(self.gate)
+
```
Comment (dabacon) [inline cirq/ops/gate_operation.py]: if you are bored, a lot of these delegates use the private _gate versus the property gate.  Would be nice to make consistent.

Comment (viathor) [reply cirq/ops/gate_operation.py]: Made them use the property instead of the private field. Thanks for spotting this.

**Comment by viathor** [comment]
Made a trivial formatting change to trigger cla/google.

**Comment by Strilanc** [comment]
Why did you need to trigger the cla check?

In case of annoyance, I have an override script.

**Comment by viathor** [comment]
It appeared to have gotten stuck... or maybe was just taking very long. I'm used to cla/google reporting its status before Travis and in this case it was still waiting T minutes after Travis completed. Don't remember the value of T, though.

**Pull request merged**